tasks/strings/strings.js

```diff
@@ -4,23 +4,38 @@
  * @param {string} str
  */
 function reverseString(str) {
+  if (typeof str != 'string') {
+    return 'This is not a string!';
+  }
+  return (str === '') ? '' : reverseString(str.substr(1)) + str.charAt(0);
```

Comment:
Please explain to me in DM how it works :)
Why have not you used split('').reverse().join('') construction?

Reply:
Here i used one of the approaches found on freeCodeCamp ( + compactness of ternary ):

https://www.freecodecamp.org/news/how-to-reverse-a-string-in-javascript-in-3-different-ways-75e4763c68cb/